name-registry/tests/functions/set_owner.rs

```diff
@@ -48,30 +48,29 @@ mod success {
 mod revert {
     use crate::utils::{
         abi::{register, set_owner},
-        setup, REGISTER_DURATION,
+        setup, Account, REGISTER_DURATION,
     };
     use fuels::prelude::*;
 
     #[tokio::test]
     #[should_panic(expected = "Revert(42)")]
     async fn cant_set_owner() {
         let (instance, _id, wallet, wallet2) = setup().await;
-        let wallet_identity = Identity::Address(Address::from(wallet.address()));
-        let name = String::from("SwaySway");
+        let acc1 = Account::new(wallet);
```

Comment:
Why not move this _into_ the `setup()` so that it gets returned as well? 1 fewer line per test